lms/djangoapps/grades/new/course_grade.py

```diff
@@ -24,8 +27,40 @@ class CourseGrade(object):
     def __init__(self, student, course, course_structure):
         self.student = student
         self.course = course
+        self.course_version = getattr(course, 'course_version', None)
+        self.course_edited_timestamp = getattr(course, 'subtree_edited_on', None)
```

Comment:
Why is it necessary to store these 2 as member variables?  Since they are simple attributes on the course, no need to cache their values in the class, right?